### PR TITLE
janitoring: Refactored dolna and horna

### PR DESCRIPTION
**Pull request by mrshu**
- Many things happened to be repeatedly used in the codebase, this
  commit refactores that (especially in functions dolna and horna) into
  a helper function.

Signed-off-by: mr.Shu mr@shu.io

### CODE REVIEW

**Comment by mrshu** [comment]
to @tWido

**Comment by pepol** [comment]
@mrshu I think the `_meals_from_bs` function should be renamed to something more explanatory, what do you think?

**Comment by mrshu** [comment]
@pepol nah, it does precisely what its name suggests, returns `Meal`s from `BeautifulSoap` strings. That's precisely it!

File: hladnymatfyzak/__init__.py [hunk 1]
```diff
@@ -39,6 +39,21 @@ def __repr__(self):
                         ")>")
 
 
+def _meals_from_bs(data, type, opts):
```
Comment (tWido) [inline hladnymatfyzak/__init__.py:42]: @mrshu : Could you just rename variable data? To something that hints it is string type, that is later used in regex. For example html_data or site_source...

**Pull request merged**